src/FormBuilder/resources/js/form-builder/src/blocks/fields/payment-gateways/PerFormSettings/index.tsx

```diff
@@ -0,0 +1,56 @@
+import {BlockEditProps} from '@wordpress/blocks';
+import {getFormBuilderWindowData} from '@givewp/form-builder/common/getWindowData';
+import Stripe from './Stripe';
+import {useEffect} from 'react';
+
+export default function PerFormSettings(props: BlockEditProps<any>) {
```

Comment:
@pauloiankoski this looks very clever! although I think it's a little too abstracted and inflexible for gateways.  This goes back to my comment regarding the gateway specific attributes instead of one big gateway settings attributes.

I think a better approach for this gateway block would be to have gateways use the [registerBlockType](https://developer.wordpress.org/block-editor/reference-guides/filters/block-filters/#blocks-registerblocktype) to add their own specific attributes and settings.  Then all their logic is isolated and lives alongside the rest of their code.  This would also give third party (and add-on) gateways an interface for their own settings.

Very similar to how we interacted with the amount block in currency switcher - by using the filter to add attributes/settings.

For the time being, I would be open to keeping the stripe specific attributes/settings in core and refactoring it later to work as a filter - but i'll leave that decision up to you.